public/app/plugins/panel/barchart/module.tsx

```diff
@@ -0,0 +1,124 @@
+import { FieldColorModeId, FieldConfigProperty, PanelPlugin, VizOrientation } from '@grafana/data';
+import { BarChartPanel } from './BarChartPanel';
+import {
+  BarChartFieldConfig,
+  BarChartOptions,
+  BarStackingMode,
+  BarValueVisibility,
+  graphFieldOptions,
+} from '@grafana/ui';
+import { addAxisConfig, addHideFrom, addLegendOptions } from '../timeseries/config';
+import { defaultBarChartFieldConfig } from '@grafana/ui/src/components/BarChart/types';
+
+export const plugin = new PanelPlugin<BarChartOptions, BarChartFieldConfig>(BarChartPanel)
+  .useFieldConfig({
+    standardOptions: {
+      [FieldConfigProperty.Color]: {
+        settings: {
+          byValueSupport: false,
+        },
+        defaultValue: {
+          mode: FieldColorModeId.PaletteClassic,
```

Comment:
hopefully, by v8 we will have a new better default palette (don't like that the first two colors are green and yellow)